models/__init__.py

```diff
@@ -0,0 +1,38 @@
+# coding=utf-8
+from sqlalchemy import func
+from models.cyberconnect2 import Session
+from models.ice_time import Ice_Time
+
+session = Session()
```

Comment:
Ideally, creating any states, like a session here, should be done during main runtime. Meaning, that it should be created once it would be used. If this module (and it's submodules) would be imported, a session would be initialized as side-effect. This can create some unexpected behaviors, or confusion, for those wanting to use it as a library. Another strong point, for moving the state creation (initialization) to consumer (main runtime) is testability. Otherwise, creation of tests would require a monkey-patching.

This is just a thought and a general recommendation. There are a usual breakers of this, like creation of a module logger.